test/exporter.js

```diff
@@ -82,7 +87,7 @@ test('Exporter.addCard', t => {
   t.is(notesUpdate[':sfld'], front);
   t.is(notesUpdate[':flds'], front + separator + back);
   t.is(notesUpdate[':mid'], topModelId);
-  t.is(notesUpdate[':csum'], checksum(front + separator + back));
+  t.is(notesUpdate[':csum'], exporter.checksum(front + separator + back));
```

Comment:
`exporter.checksum('some string'), 2336613565` That style from above would be more appropriate here, instead of `exporter.checksum(front + separator + back)`, just the generated number